scripts/sct_label_vertebrae.py

```diff
@@ -239,83 +237,82 @@ def main(args=None):
-    printv('\nClean labeled segmentation (correct interpolation errors)...', verbose)
+    sct.printv('\nClean labeled segmentation (correct interpolation errors)...', verbose)
     clean_labeled_segmentation('segmentation_labeled.nii.gz', 'segmentation.nii.gz', 'segmentation_labeled.nii.gz')
 
     # label discs
-    printv('\nLabel discs...', verbose)
+    sct.printv('\nLabel discs...', verbose)
     label_discs('segmentation_labeled.nii.gz', verbose=verbose)
 
     # come back to parent folder
     os.chdir('..')
 
     # Generate output files
-    path_seg, file_seg, ext_seg = extract_fname(fname_seg)
-    printv('\nGenerate output files...', verbose)
-    generate_output_file(path_tmp+'segmentation_labeled.nii.gz', path_output+file_seg+'_labeled'+ext_seg)
-    generate_output_file(path_tmp+'segmentation_labeled_disc.nii.gz', path_output+file_seg+'_labeled_discs'+ext_seg)
+    path_seg, file_seg, ext_seg = sct.slash_at_the_end(fname_seg)
+    sct.printv('\nGenerate output files...', verbose)
+    sct.generate_output_file(path_tmp+'segmentation_labeled.nii.gz', path_output+file_seg+'_labeled'+ext_seg)
+    sct.generate_output_file(path_tmp+'segmentation_labeled_disc.nii.gz', path_output+file_seg+'_labeled_discs'+ext_seg)
     # copy straightening files in case subsequent SCT functions need them
-    generate_output_file(path_tmp+'warp_curve2straight.nii.gz', path_output+'warp_curve2straight.nii.gz', verbose)
-    generate_output_file(path_tmp+'warp_straight2curve.nii.gz', path_output+'warp_straight2curve.nii.gz', verbose)
-    generate_output_file(path_tmp+'straight_ref.nii.gz', path_output+'straight_ref.nii.gz', verbose)
+    sct.generate_output_file(path_tmp+'warp_curve2straight.nii.gz', path_output+'warp_curve2straight.nii.gz', verbose)
+    sct.generate_output_file(path_tmp+'warp_straight2curve.nii.gz', path_output+'warp_straight2curve.nii.gz', verbose)
+    sct.generate_output_file(path_tmp+'straight_ref.nii.gz', path_output+'straight_ref.nii.gz', verbose)
 
     # Remove temporary files
     if remove_tmp_files == 1:
-        printv('\nRemove temporary files...', verbose)
-        run('rm -rf '+path_tmp)
+        sct.printv('\nRemove temporary files...', verbose)
+        sct.run('rm -rf '+path_tmp)
 
     # to view results
-    printv('\nDone! To view results, type:', verbose)
-    printv('fslview '+fname_in+' '+path_output+file_seg+'_labeled'+' -l Random-Rainbow -t 0.5 &\n', verbose, 'info')
+    sct.printv('\nDone! To view results, type:', verbose)
+    sct.printv('fslview '+fname_in+' '+path_output+file_seg+'_labeled'+' -l Random-Rainbow -t 0.5 &\n', verbose, 'info')
 
 
 # Detect vertebral levels
 # ==========================================================================================
-def vertebral_detection(fname, fname_seg, contrast, init_disc=[], verbose=1, path_template='', initc2='auto'):
+def vertebral_detection(fname, fname_seg, contrast, init_disc=[], verbose=1, path_template='', initc2='auto',
```

Comment:
Why do we need more parameters for vertebral_detection? I want to avoid changing too much the function properties until we migrate into a modular structure.